### PR TITLE
[4385] Improve message list screen docs

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### 🎯 Goal

Small improvement for https://github.com/GetStream/stream-chat-android/pull/4390

screencapture-localhost-3000-chat-docs-sdk-android-ui-message-components-message-list-screen-2022-11-11-17_22_20

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx [hunk 1]
```diff
@@ -61,8 +61,11 @@ class MyMessageListActivity : AppCompatActivity(R.layout.fragment_container) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         if (savedInstanceState == null) {
+            val fragment = MessageListFragment.newInstance(cid = "messaging:123") {
+                showHeader(true)
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx:65]: Hidden by default, we need to set this to "true" to be on par with the example above.

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/MessageListActivity.kt [hunk 2]
```diff
@@ -63,10 +63,23 @@ public open class MessageListActivity : AppCompatActivity() {
...
+         */
         @JvmStatic
         @JvmOverloads
-        public fun createIntent(context: Context, cid: String, messageId: String? = null): Intent {
-            return Intent(context, MessageListActivity::class.java).apply {
+        public fun createIntent(
+            context: Context,
+            cid: String,
+            messageId: String? = null,
+            activityClass: Class<out MessageListActivity> = MessageListActivity::class.java
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/MessageListActivity.kt:80]: Not the best API to pass the concrete class, but should be fine taking into account the the Activity is mostly used for a quick start and shouldn't be used in production by the clients.

File: docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx [hunk 3]
```diff
@@ -133,11 +136,7 @@ public final class MyMessageListActivity extends AppCompatActivity implements Me
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        if (savedInstanceState == null) {
-            getSupportFragmentManager().beginTransaction()
-                .replace(R.id.container, MessageListFragment.newInstance("messaging:123"))
-                .commit();
-        }
+        // Add MessageListFragment to the layout
```
Comment (MarinTolic) [inline docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx:139]: Down on line 151 I would suggest exchanging

```
Currently, there is one click listener you can use with the `MessageListFragment`:
```

with

```
Currently, there's only a single click listener you can use with the `MessageListFragment`:
```

Comment (MarinTolic) [inline docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx:139]: Down on line 151 I would suggest exchanging

```
Currently, there is one click listener you can use with the `MessageListFragment`:
```

with

```
Currently, there's only a single click listener you can use with the `MessageListFragment`:
```

**Review by MarinTolic — APPROVED** [review]

**Pull request merged**